docs/rfcs/merger_matcher/README.md

```diff
@@ -78,32 +93,25 @@ For example creating a combined work 'A-B' when processing work 'A' could give a
 }
 ```
 Note that `A.identifiers` includes the source identifier for  `A` itself.
-In the case that neither A or B has previously been identified as a part of a combined work the matcher/redirector 
-will output: 
-```javascript
-redirects = [ 
-  { 
-    target: 'sierra-system-number/s12345+miro-image-number/s67890' // A-B
-    sources: [ 'sierra-system-number/s12345', 'miro-image-number/s67890'] // A, B
-  }
-]
+In the case that neither A or B has previously been identified as a part of a combined work the matcher
+will output:
+```[json]
+{
+  "work-groups": [
+    {
+      "identifiers": [ "sierra-system-number/s12345", "miro-image-number/s67890"]
+    }
+  ]
+}
 ```
```

Comment:
Perhaps output:




```[json]
{
  "linked-works": [
    {
      "identifiers": [ "sierra-system-number/s12345", "miro-image-number/s67890"]
    }
  ]
}

Reply:
You mean if the merger needs to know which work caused the update? I don't think so, but let's see what the others think